automerge-protocol/Cargo.toml

```diff
@@ -8,6 +8,8 @@ edition = "2018"
 
 [dependencies]
 hex = "^0.4.2"
+sha2 = "^0.8.1"
```

Comment:
Do these still need to be here now that we've moved the encoding back to the backend crate?

Reply:
good catch